Tests/UploadTests.swift

```diff
@@ -518,6 +518,32 @@ class UploadMultipartFormDataTestCase: BaseTestCase {
             XCTFail("Content-Type header value should not be nil")
         }
     }
+    
+    func testThatUploadingMultipartFormDataWithNotExistFileShouldFail() {
```

Comment:
```suggestion
    func testThatUploadingMultipartFormDataWithNonexistentFileThrowsAnError() {
```

This can be made a bit easier to read.